plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -45,16 +151,35 @@ async fn main() -> Result<(), anyhow::Error> {
     // Mqtt message loop
     while let Some(message) = mqtt_client.received.next().await {
         debug!("Received {:?}", message);
-        match message.payload_str()?.split(',').nth(0).unwrap_or_default() {
-            "524" => {
-                debug!("{}", message.payload_str()?);
-                todo!() // c8y_DownloadConfigFile
-            }
-            "526" => {
-                debug!("{}", message.payload_str()?);
-                todo!() // c8y_UploadConfigFile
+        if let Ok(payload) = message.payload_str() {
+            let result = match payload.split(',').next() {
```

Comment:
I prefer 
```suggestion
            let result = match payload.split(',').next().unwrap_or_default() {
```
then, remove `Some()` from the conditions.